### PR TITLE
Fix NPD CI job setup

### PR DESCRIPTION
**Pull request by wangzhen127**
Fix NPD CI job setup

### CODE REVIEW

File: config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml [hunk 1]
```diff
@@ -33,8 +34,12 @@ periodics:
       command:
       - runner.sh
       args:
-      - make
-      - test
+      - bash
+      - -c
+      - |
+        apt-get update &&
+        apt-get install -y libsystemd-dev &&
```
Comment (Random-Liu) [inline config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml]: Can we add a command in the script for this? I don't think we want to have this logic in 2 different places.

Comment (wangzhen127) [reply config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml]: Sure. I can do that.

Comment (wangzhen127) [reply config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml]: Updated.

**Pull request merged**